VideoAPI/VideoApi.UnitTests/Controllers/ConferenceManagement/TransferParticipantTests.cs

```diff
@@ -0,0 +1,80 @@
+using System;
+using System.Net;
+using System.Threading.Tasks;
+using FluentAssertions;
+using Microsoft.AspNetCore.Mvc;
+using Moq;
+using NUnit.Framework;
+using VideoApi.Contract.Requests;
+using VideoApi.Domain.Enums;
+using VideoApi.Services.Kinly;
+
+namespace VideoApi.UnitTests.Controllers.ConferenceManagement
+{
+    public class TransferParticipantTests : ConferenceManagementControllerTestBase
+    {
+        [Test]
+        public async Task should_move_participant_into_hearing_room()
+        {
+            var conferenceId = Guid.NewGuid();
+            var request = new TransferParticipantRequest
+            {
+                ParticipantId = Guid.NewGuid(),
+                TransferType = TransferType.Call
+            };
+            
+            var result = await Controller.TransferParticipantAsync(conferenceId, request);
+            var typedResult = (OkResult) result;
+            typedResult.Should().NotBeNull();
+            typedResult.StatusCode.Should().Be((int) HttpStatusCode.OK);
+            
+            VideoPlatformServiceMock.Verify(
+                x => x.TransferParticipantAsync(conferenceId, request.ParticipantId, RoomType.WaitingRoom,
+                    RoomType.HearingRoom), Times.Once);
+        }
+        
+        [Test]
+        public async Task should_dismiss_participant_from_hearing_room()
+        {
+            var conferenceId = Guid.NewGuid();
+            var request = new TransferParticipantRequest
+            {
+                ParticipantId = Guid.NewGuid(),
+                TransferType = TransferType.Dismiss
+            };
+            
+            var result = await Controller.TransferParticipantAsync(conferenceId, request);
+            var typedResult = (OkResult) result;
+            typedResult.Should().NotBeNull();
+            typedResult.StatusCode.Should().Be((int) HttpStatusCode.OK);
+
+            VideoPlatformServiceMock.Verify(
+                x => x.TransferParticipantAsync(conferenceId, request.ParticipantId, RoomType.HearingRoom,
+                    RoomType.WaitingRoom), Times.Once);
+        }
+        
+        [Test] public async Task should_return_kinly_status_code_on_error()
+        {
+            var conferenceId = Guid.NewGuid();
+            var request = new TransferParticipantRequest
+            {
+                ParticipantId = Guid.NewGuid(),
+                TransferType = TransferType.Call
+            };
+            var message = "Auto Test Error";
+            var response = "You're not allowed to start this hearing";
```

Comment:
wrong error? doesnt matter i guess but would make it easier to understand the test case if it was relevant maybe?

Reply:
Updated. Thanks this was a poor copy and paste job